wpeverest/everest-forms
# Feature - Delete option in Header logo in PDF Submission

### All Submissions:






### Changes proposed in this Pull Request:
This PR adds option for deleting the image uploaded previously in the PDF submission header.



Closes # .

### How to test the changes in this Pull Request:

1. Goto globlal-setting in PDF submission add-on.
2.  Try to upload the header logo. Hover over the uploaded image , we can see delete button.
3.  Click to delete and save. 

### Types of changes:




### Other information:




### Changelog entry

Feature - Delete option in Header image in PDF Submission
